CMakeLists.txt

```diff
@@ -77,7 +90,9 @@ if(${YOMM2_ENABLE_DOC})
     DEPENDS "${readme_cpp}")
 endif()
 
-add_subdirectory(docs.in)
+if(${YOMM2_ENABLE_TESTS} OR YOMM2_ENABLE_DOC)
```

Comment:
If docs are enabled but not tests, this will still generate some test binaries because down the tree (e.g. https://github.com/jll63/yomm2/blob/master/docs.in/reference/CMakeLists.txt), test executables are not wrapped inside `if(YOMM2_ENABLE_TESTS) ... endif()`. Not sure if it is intended

Reply:
It is. `YOMM2_ENABLE_DOC` is not documented (haha). It is for my usage only. However, both the examples in the docs and in the `examples` directories are used as tests. In `vcpkg` context you explicitly [set tests to off](https://github.com/microsoft/vcpkg/blob/master/ports/yomm2/portfile.cmake#L23) so we should be fine...